cypress/fixtures/mockPosts.json

```diff
@@ -0,0 +1,54 @@
+{
+  "posts": [
```

Comment:
Ah this one looks closer to the graphQL signature, I would just double check before stubbing/testing